src/pages/K8s.vue

```diff
@@ -158,7 +155,11 @@ export default {
       that.$data.state = stt;
     });
     ipcRenderer.on('settings-update', (event, settings) => {
-      this.$data.settings = settings;
+      // TODO: put in a status bar
+      console.log('settings have been updated');
+
+      // NOOOOO:!
```

Comment:
Please elaborate; _why_ is this a no?

Reply:
This was my knee-jerk response from when the sliders were free-form fields, but I'd like to keep the model like so:
* when a form is shown, its fields are initialized based on prefs and current state
* field values change only by user action
* if saving a field requires making some change when saving it, but there's no need to change how it's stored in the field, don't change it under the user's feet.

I'll re-evaluate that with the current set of widgets.

Reply:
I'd be happy with a similarly verbose comment in the code.

Reply:
Turns out we can currently update the prefs on update with no problems. This was the behavior when the only setting changed the version field. So let's leave it that way until something breaks.